contents/handbook/why-does-posthog-exist.md

```diff
@@ -10,18 +10,25 @@ Equip every developer to build successful products
 
 ## Our strategy
 
-### Provide every tool needed to build successful products
+### 1. Provide every tool needed to build successful products
 
-Building a successful product is hard. It gets harder when you don't understand how your customers are using your product.
+Building a successful product is hard; doing so when you don't understand your customers is even harder.
 
-PostHog includes all the tools you need to understand how people use your product. All with one simple snippet.
+PostHog includes every the tool you need to understand how people use your product, who they are, and how to make them happy.
 
-### Start from the start
+### 2. Useful at every stage
```

Comment:
Is this something that was discussed somewhere / agreed to change? I think this framing of "start from the start" is useful strategically, whereas "useful at every stage" is less specific and generic and not very helpful.

Reply:
No, just my suggestion. "Start from the start" is just confusing to me! It think this used by "Get in first", which made more sense to me. Anyway, I don't feel strongly, so can remove this.

Reply:
I agree, I like the prior wording to "get in first" better